src/test/java/uk/gov/pay/ledger/transaction/service/AccountIdSupplierManagerTest.java

```diff
@@ -65,7 +66,7 @@ public void givenAccountIdNotProvidedWhenNotRequiredUsesPrivilegedSupplier() {
     }
 
     private void setUpAndGet(Boolean overrideAccountRestriction, String gatewayAccountId) {
-        new AccountIdSupplierManager(overrideAccountRestriction, gatewayAccountId)
+        new AccountIdListSupplierManager(overrideAccountRestriction, gatewayAccountId == null ? null : List.of(gatewayAccountId))
```

Comment:
Doesn’t `AccountIdListSupplierManager` handle `gatewayAccountIds` being null?

Reply:
this is really annoying me, can't work out to fix it atm, it works, something about the mocks, anyway